### DISCUSSION TITLE
Beatmap info overlay displaying classic max combo for lazer scores

### DISCUSSION THREAD

**Discussion by Loreos7** [ppy/osu#19321] [Q&A] [answered]
For none-classic mode fractional tooltip shows the same max combo values as for classic mode.

https://user-images.githubusercontent.com/86934170/180600481-df3105cd-63aa-4f53-8a4e-e037162c6cef.mp4

**Comment by frenzibyte** [comment, marked as answer]
That's the current behaviour of the overlay, it doesn't consider different max combo methods yet.

**Reply by frenzibyte** [reply]
Will be resolved in https://github.com/ppy/osu/pull/19344
